main.html

```diff
@@ -1,47 +1,72 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-<meta charset="UTF-8">
-<title>Page Title</title>
-<link rel="stylesheet" href="main.css">
-<link rel="stylesheet" href="normalize.css">
+    <meta charset="UTF-8">
+    <meta content='width=device-width, initial-scale=1.0, user-scalable=0' name='viewport'/>
+    <title>Page Title</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css">
+    <link rel="stylesheet" href="main.css">
 </head>
 <body>
     <header>
         <div class="header">
             <figure class="header__image">
```

Comment:
<img width="248" alt="Screen Shot 2022-06-25 at 1 03 20 PM" src="https://user-images.githubusercontent.com/18077388/175788958-559abd5a-aa5b-4d8f-93a1-c5aaa02dac23.png">
use img tag for raster images and svg for vector images, I'd also double-check the size because 5x5 looks weird